mathics/eval/numbers.py

```diff
@@ -1,10 +1,109 @@
+from typing import Optional
+
+import mpmath
 import sympy
 
+from mathics.core.atoms import Complex, MachineReal, PrecisionReal, Real
 from mathics.core.convert.sympy import from_sympy
+from mathics.core.element import BaseElement
 from mathics.core.expression import Expression
+from mathics.core.number import MACHINE_PRECISION_VALUE, ZERO_MACHINE_ACCURACY, dps
 from mathics.core.symbols import SymbolPlus
 
 
+def eval_accuracy(z: BaseElement) -> Optional[float]:
```

Comment:
We seem to have started to have two conventions now for the naming of the `eval_` methods here. There are some that keep lower case like this here. And some that use the uppercase of the builtin name like `eval_N`, `eval_Plot`. 

We should decide on a convention one way or another and stick to it. I have a slight preference for using the uppercase letter because it makes the connection between the Mathics3 built-in name and the particular function. If it is a little unconventional from the normal Python naming, that might be a good thing in order to make the developer reading the code that this has a special function.

But more important is to be consistent. Your thoughts?

Reply:
Ok - it looks like we are going with `eval_Accuracy`.  Thanks!  I think this is the right thing. 

I will resolve this but note there are other `eval_` functions that need adjusting: `evall_fulform_makeboxes`, `eval__makeboxes`.  and `eval_p_norm`. That can be done in another PR.